airbyte-integrations/connectors/source-facebook-marketing/unit_tests/test_migrations/test_new_config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "start_date": "2021-02-08T00:00:00Z",
+  "end_date": "2021-02-15T00:00:00Z",
+  "custom_insights": [
+    {
+      "name": "custom_insight_stream",
+      "fields": ["account_name", "clicks", "cpc", "account_id", "ad_id"],
+      "breakdowns": ["gender"],
+      "action_breakdowns": []
+    }
+  ],
+  "account_id": "01234567890",
```

Comment:
the new config should not have this still, right?

Reply:
also, nit: I would name this file test_config and the other one test_old_config - it is clearer migration wise and in future versions it is clear what is the current state of the config

Reply:
Yes, `account_id` would be in the config if it is migrated; otherwise, only `account_ids`.

Reply:
got it. in this case i think 3 test files could be here - one with account_id (previous version), one with account_id and account ids (upgraded from previous version), one with account_ids only (created or reset on new version)

Reply:
I added one more testing config.